[2021-08-06 09:24:02,297] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: ex_dag_v_1_0.task_b 2021-08-06T09:23:00+00:00 [queued]>
[2021-08-06 09:24:02,310] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: ex_dag_v_1_0.task_b 2021-08-06T09:23:00+00:00 [queued]>
[2021-08-06 09:24:02,310] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-06 09:24:02,310] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-06 09:24:02,310] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-06 09:24:02,317] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_b> on 2021-08-06T09:23:00+00:00
[2021-08-06 09:24:02,321] {standard_task_runner.py:52} INFO - Started process 8421 to run task
[2021-08-06 09:24:02,326] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_dag_v_1_0', 'task_b', '2021-08-06T09:23:00+00:00', '--job-id', '53', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ex_dag_v_1_0.py', '--cfg-path', '/tmp/tmpqvdmrfg_', '--error-file', '/tmp/tmptve8ucl9']
[2021-08-06 09:24:02,326] {standard_task_runner.py:77} INFO - Job 53: Subtask task_b
[2021-08-06 09:24:02,369] {logging_mixin.py:104} INFO - Running <TaskInstance: ex_dag_v_1_0.task_b 2021-08-06T09:23:00+00:00 [running]> on host a3e01b137881
[2021-08-06 09:24:02,420] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Nha
AIRFLOW_CTX_DAG_ID=ex_dag_v_1_0
AIRFLOW_CTX_TASK_ID=task_b
AIRFLOW_CTX_EXECUTION_DATE=2021-08-06T09:23:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-06T09:23:00+00:00
[2021-08-06 09:24:02,421] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-06 09:24:02,422] {subprocess.py:63} INFO - Running command: ['bash', '-c', "echo 'task_b'"]
[2021-08-06 09:24:02,433] {subprocess.py:75} INFO - Output:
[2021-08-06 09:24:02,435] {subprocess.py:79} INFO - task_b
[2021-08-06 09:24:02,435] {subprocess.py:83} INFO - Command exited with return code 0
[2021-08-06 09:24:02,458] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=ex_dag_v_1_0, task_id=task_b, execution_date=20210806T092300, start_date=20210806T092402, end_date=20210806T092402
[2021-08-06 09:24:02,479] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-06 09:24:02,496] {local_task_job.py:151} INFO - Task exited with return code 0
